src/services/api/AxiosInstance.ts

```diff
@@ -25,9 +27,11 @@ const requestFormatter = async (axiosConfig: AxiosRequestConfig) => {
     authMessage === "token undefined"
 
   if (isEmailLoginUser) {
-    const accessToken = await getAccessToken()
-    axiosConfig.headers = {
-      Authorization: `token ${accessToken}`,
+    const accessToken = await tokenServiceInstance
+      .getAccessToken()
+      .unwrapOr("null")
+    if (axiosConfig.headers) {
+      axiosConfig.headers.Authorization = `token ${accessToken}`
```

Comment:
From eslint: Assignment to property of function parameter 'axiosConfig'

Reply:
Changed to create new object with Authorization instead of assignment.